src/Dofs/DofHandler2.jl

```diff
@@ -0,0 +1,225 @@
+
+# struct Field
+#     name::Symbol
+#     interpolation::Interpolation
+#     dim::Int
+# end
+
+mutable struct DofHandler{dim,N,T,M}
```

Comment:
What happend to `dofs_nodes::Matrix{Int}` (aka DOF in calfem-world)

Reply:
That thing doesn't actually make much sense. Nothing says that all nodes need a dof, nothing says that there are no dofs at locations with no nodes, and nothing says that all nodes have equal number of dofs.

Dofs are something that is defined on the element level. When exporting data for visualization, it is common to give the field values in the nodes, but that is a post processing step.

Reply:
I thought dofs was defined on nodes, not on the elements, since we get the results in the nodes

Reply:
I thought dofs was defined on nodes, not on the elements, since we get the results in the nodes